src/articles/FindArticles.js

```diff
@@ -80,10 +80,11 @@ export default function NewArticles({ api }) {
         originalList={originalList}
         setArticleList={setArticleList}
       />
-      <Reminder hasExtension={hasExtension} ></Reminder>
+      <Reminder hasExtension={hasExtension}></Reminder>
       {articleList.map((each) => (
         <ArticlePreview key={each.id} article={each} api={api} />
       ))}
+      <NoArticles articleList={articleList}></NoArticles>
```

Comment:
Can we rename this to something more descriptive of what the component does? e.g. `ShowLinkRecommendationsIfNoArticles`? Or something like that?